tools/cdep/commands/update_default.go

```diff
@@ -5,6 +5,13 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/cuvva/cuvva-public-go/lib/config"
+	"github.com/cuvva/cuvva-public-go/lib/ptr"
+	"github.com/pkg/errors"
+
```

Comment:
No newline here I don't think :) All the GitHub imports get sorted together - including the "local" ones